apis/core/v1alpha1/identifiers.go

```diff
@@ -26,14 +30,34 @@ type AWSIdentifiers struct {
 	AdditionalKeys map[string]string `json:"additionalKeys,omitempty"`
 }
 
-// TargetKubernetesResource provides all the values necessary to identify a given ACK type
-// and override any metadata values when creating a resource of that type.
-type TargetKubernetesResource struct {
-	// +kubebuilder:validation:Required
-	Group string `json:"group"`
-	// +kubebuilder:validation:Required
-	Kind     string             `json:"kind"`
-	Metadata *PartialObjectMeta `json:"metadata,omitempty"`
+// NamespacedTargetKubernetesResource provides all the values necessary to identify an ACK
+// resource of a given type (within the same namespace).
+type NamespacedTargetKubernetesResource struct {
+	metav1.GroupKind `json:""`
+	Name             *string `json:"name"`
+}
+
+// AdoptedResourceTarget provides the values necessary to create a
+// Kubernetes resource and override any of its metadata values.
+type AdoptedResourceTarget struct {
+	metav1.GroupKind `json:""`
+	Metadata         *PartialObjectMeta `json:"metadata,omitempty"`
+}
+
+// ResourceFieldSelector provides the values necessary to identify an individual
+// path on an individual K8s resource.
+type ResourceFieldSelector struct {
+	Resource NamespacedTargetKubernetesResource `json:"resource"`
+	Path     *string                            `json:"path"`
+}
+
+// FieldExportOutputSelector provides the values necessary to identify the
+// output path for a field export.
+type FieldExportOutputSelector struct {
```

Comment:
Now, this particular struct *is* specific to a FieldExport struct, so I would move it to the `field_export.go` file and I'd recommend renaming it `FieldExportTarget` because that's what it describes.